pymc3/sampling.py

```diff
@@ -481,6 +490,11 @@ def sample(draws=500, step=None, init='auto', n_init=200000, start=None, trace=N
 
     return trace
 
+def sample(*args,**kwargs):
+    """This method is deprecated.  Please use :func:`~sampling.sample_posterior`"""
```

Comment:
Actually I don't think we want to deprecate this function, just because it's the main entry point for every user. Instead, I would propose to make `sample_posterior` an alias to `sample()`. CC @junpenglao @fonnesbeck

Reply:
Agree.

Reply:
Ah, sorry about that.  Will change.